src/packages/bem/__tests__/bem.spec.js

```diff
@@ -37,6 +37,16 @@ describe('BEM decorator', () => {
             expect(buttonLabel.hasClass('Button__label--size_2')).toBe(true);
         });
 
+        it('should add "wildcard" class names for modifier with value (striong or number)', () => {
```

Comment:
might be good to validate that the classname with the value is added too in case if it is defined? ex: `Button__icon--size-2`

Reply:
This behavior is covered in the previous test case "should add proper class names based on propsToMods and passed props"